docs/en_US/TrainingService/PaiMode.md

```diff
@@ -190,4 +205,4 @@ Check policy:
 3. Note that the version check feature only check first two digits of version.For example, NNIManager v0.6.1 could use trialKeeper v0.6 or trialKeeper v0.6.2, but could not use trialKeeper v0.5.1 or trialKeeper v0.7.
 
 If you could not run your experiment and want to know if it is caused by version check, you could check your webUI, and there will be an error message about version check.
-![](../../img/version_check.png)
+![](../../img/version_check.png)
```

Comment:
Looks like this change should be revert. Please help check.
![image](https://user-images.githubusercontent.com/61399850/99209115-5a4ae900-27fd-11eb-96cc-fb3c4b7e4292.png)